package.json

```diff
@@ -41,6 +41,7 @@
     "mocha-lcov-reporter": "^1.2.0",
     "mock-spawn": "^0.2.1",
     "mockery": "~1.4.0",
+    "nightwatch-component-a": "git://github.com/cesine/nightwatch-component-a",
```

Comment:
yeah i agree,  it would be great to be able to test this, but not rely on an external package. 

I set up a full example here which is external to nightwatch, and is how I debugged this PR https://github.com/cesine/nightwatch-modular-example